doc/source/comparison_with_r.rst

```diff
@@ -530,6 +530,103 @@ For more details and examples see :ref:`categorical introduction <categorical>`
 :ref:`differences to R's factor <categorical.rfactor>`.
 
 
```

Comment:
If you aren't familiar with sphinx, you need to start the line with a `..` http://www.sphinx-doc.org/en/stable/markup/inline.html#cross-referencing-arbitrary-locations